qlib/data/data.py

```diff
@@ -638,14 +648,9 @@ def feature(self, instrument, field, start_index, end_index, freq):
         # validate
         field = str(field).lower()[1:]
         instrument = code_to_fname(instrument)
-        uri_data = self._uri_data.format(instrument.lower(), field, freq)
-        if not os.path.exists(uri_data):
-            get_module_logger("data").warning("WARN: data not found for %s.%s" % (instrument, field))
-            return pd.Series(dtype=np.float32)
-            # raise ValueError('uri_data not found: ' + uri_data)
-        # load
-        series = read_bin(uri_data, start_index, end_index)
-        return series
+
+        self.backend.setdefault("kwargs", {}).update(instrument=instrument, field=field, freq=freq)
```

Comment:
If we have to set the config for each backend_obj, why don't use the config as the parameters of the function?